compatibility/bazel_tools/client_server/WithPostgres.hs

```diff
@@ -0,0 +1,82 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+module Main (main) where
+
+import qualified Bazel.Runfiles as Runfiles
+import Control.Exception
+import Data.Text (Text)
+import qualified Data.Text as T
+import System.Directory.Extra
+import System.Environment
+import System.FilePath
+import System.IO.Extra
+import System.Process
+
+-- This is modelled after com.daml.testing.postgresql.PostgresAround.
+-- We make this a separate executable since it should only
+-- depend on released artifacts so we cannot easily use sandbox as a library.
```

Comment:
We could, in theory, release this with a `main` function, but this version seems nicer so I'm inclined to say we should not do that.

Reply:
right there are ways to make this work but I think having it be a separate process is actually a good idea.